weave/trace_server/trace_server_converter.py

```diff
@@ -91,6 +91,8 @@ def replace_ref(ref_str: str) -> str:
                 project_id
             )
         external_project_id = int_to_ext_project_cache[project_id]
+        if not external_project_id:
+            return f"{ri.WEAVE_SCHEME}://///{tail}"
```

Comment:
this is not correct - it should not be `WEAVE_SCHEME` because we have empty project/name

Reply:
oooh I misread your previous comment, now i see `weave-private:`...